default.xml

```diff
@@ -32,5 +32,5 @@
   <project name="openembedded/meta-linaro" path="layers/meta-linaro" remote="linaro" revision="master"/>
   <project name="openembedded/meta-openembedded" path="layers/meta-openembedded" remote="github"/>
   <project name="openembedded/openembedded-core" path="layers/openembedded-core" remote="github"/>
-  <project name="petegriffin/meta-wpe" path="layers/meta-wpe" remote="github" revision="54231d6ed6787c539334bfb759fb0aadc27866d8"/>
+  <project name="petegriffin/meta-wpe" path="layers/meta-wpe" remote="github"/>
```

Comment:
The ci build failure mentioned in the commit description is:
`16:47:39 ERROR: Layer wpe-layer is not compatible with the core layer which only supports these series: dunfell (layer is compatible with sumo thud zeus warrior)`
When merged into lhg-oe-manifests, master branch, this change starts using petegriffin/meta-wpe, master branch for meta-wpe. But looking at the https://github.com/petegriffin/meta-wpe/blob/master/conf/layer.conf#L12 , petegriffin/meta-wpe isn't dunfell compatible...

The change itself looks OK, so I am approving it,
But petegriffin/meta-wpe, master branch needs to be updated to fix the jenkins build failure for real